docs/supported-languages.md

```diff
@@ -76,10 +137,123 @@ The following thresholds define each maturity level:
 
 Visit the cheat sheet generation script and associated semgrep-core test files to learn more about each feature:
 
-* [generation script](https://github.com/returntocorp/semgrep/blob/develop/scripts/generate_cheatsheet.py)
-* [semgrep-core test files](https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/tests)
-## Language parse rates
+* [Generation script](https://github.com/returntocorp/semgrep/blob/develop/scripts/generate_cheatsheet.py)
+* [`semgrep-core` test files](https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/tests)
+
+### Language parse rates
+
+See [Parse rates by language](https://dashboard.semgrep.dev/).
+
+## Semgrep Supply Chain
+
+<SscIntro/>
+
+Semgrep Supply Chain parses **lockfiles** for dependencies, then scans your codebase for reachable findings based on the lockfiles. Some languages, such as Java, have several lockfiles, depending on your repository's package manager. For some languages, such as Javascript and Python, a manifest file is also parsed.
+
+The following tables list lockfiles, their package manager and language, and their support levels.
+
+### General Availability
+
```

Comment:
It might be better to state the obvious for the ease of reading the table:

```suggestion

This table provides information about fully supported (generally available (GA)) languages, specific package managers, and their lockfiles in Semgrep Supply Chain:

```